src/parser/jobs/smn/modules/Swiftcast.js

```diff
@@ -8,5 +10,48 @@ const MISSED_SEVERITIES = {
 }
 
 export default class Swiftcast extends SwiftcastModule {
+	static handle = 'swiftcast'
+
 	severityTiers = MISSED_SEVERITIES
+	_dwtActive = false
+	_dwtActiveTimestamp = 0
+
+	constructor(...args) {
+		super(...args)
+
+		this.addEventHook('cast', {to: 'player'}, this._onCast)
+		this.addEventHook('normaliseddamage', {by: 'player', abilityId: this.data.actions.DEATHFLARE.id}, this._endDWT)
+		this.addEventHook('death', {to: 'player'}, this._endDWT)
+	}
+
+	_onCast(event) {
+		const guid = event.ability.guid
+
+		if (guid === this.data.actions.DREADWYRM_TRANCE.id) {
+			this.debug(`dwt now active ${this.parser.formatTimestamp(event.timestamp)}`)
```

Comment:
Why are all the other `DWT` in caps?

Reply:
Pretty sure it was like midnight when I *started* work on this. Might have been crazy (my PR messages tend to devolve into the above when tired/crazy). Also still crazy, so I'll change.